### DISCUSSION TITLE
Help with wiring...the touch pins...ESP32 ILI9341

### DISCUSSION THREAD

**Discussion by bcodermicro** [Bodmer/TFT_eSPI#909] [Q&A - General] [answered]
So i'm a bit confuses as where to put my T_OUT and T_IN and T_CLK pins. 

I have other pins wired to: however not sure where to connect the above pins to.. 

```
#define TFT_MISO 19
#define TFT_MOSI 23
#define TFT_SCLK 18
#define TFT_CS   15  // Chip select control pin
#define TFT_DC    2  // Data Command control pin
#define TFT_RST   4  // Reset pin (could connect to RST pin)

#define TOUCH_CS 21     // Chip select pin (T_CS) of touch screen#define TFT_MISO 19

```

**Comment by Bodmer** [comment, marked as answer]
Pin names vary, these are the typical connections:
```
T_DO  -> MISO pin (e.g. TFT_MISO)
T_DIN -> MOSI pin (e.g. TFT_MOSI)
T_CS  -> pin of you choice specified in setup file as TOUCH_CS
T_CLK -> SCLK pin (e,g, TFT_SCLK)
T_IRQ -> not connected
```

**Comment by bcodermicro** [comment]
Thanks....was able to figure it out...what about including this in the Setup file? as a guide....

**Comment by Bodmer** [comment]
OK, yes it does need to be put somwhere but there are a lot of setup files so maybe the touch examples are a better place. Of course a User Manual would be even better!
